faqs/galaxy/support_messaging_system.md

```diff
@@ -0,0 +1,108 @@
+---
+title: How can I use the Matrix messaging system for Galaxy Project communication?
+area: support
+description: When you are directed to use Matrix for a Galaxy working group or for your support question, but you don't know what Matrix or how to access it. Learn what Matrix is, why it is used, and how to use Matrix to connect to us.
+box_type: tip
+layout: faq
+contributors: [hujambo-dunia]
+---
+
+### Introduction
+
+The Galaxy community uses [Matrix](https://matrix.org/) – a secure, decentralized, open-source messaging protocol – as its primary chat system. Matrix enables real-time communication across Galaxy contributors, developers, trainers, and users. Common uses include:
+
+* Getting help from the community
+* Discussing tool and workflow development
+* Organizing training events and materials
+* Collaborating on Galaxy infrastructure or scientific questions
+
+Matrix provides an open alternative to proprietary platforms like Slack and Microsoft Teams, with broader flexibility and user privacy in mind.
+
+---
+
+### 🔰 For Newbies: Getting Started with Matrix
+
+If you are new to messaging systems, here's how to get started with Galaxy on Matrix:
+
+<img src="images/support_matrix_element.jpg" alt="Galaxy Project Matrix space using the Element client app" width="350" align="right">
+
+#### Step 1: Choose and install a Matrix app
+
+**Install a Matrix client app:** Download and install a Matrix client:
+
+* [**Element**](https://element.io/) – Recommended. Available on [Web](https://app.element.io), [iOS](https://apps.apple.com/app/element/id1083446067), [Android](https://play.google.com/store/apps/details?id=im.vector.app), [Windows/macOS/Linux](https://element.io/download).&#x20;
+
+* [**Cinny**](https://cinny.in/)
+
+* [**Other Matrix clients...**](https://matrix.org/ecosystem/clients/)
+
+All of these apps connect to the same Galaxy channels. Choose the interface that works best for you.
+
+#### Step 2: Create a Matrix account
+
+* Open the app and sign up. It's easiest to use the default public server at `matrix.org` for your account.
+* Choose a username. Your Matrix ID will look like: `@yourname:matrix.org`.
+
+#### Step 3: Join the Galaxy **Lobby**
+
+* In your Matrix client, join the main Galaxy chat room.
+  * If you're using the Element client, click Explore (the compass icon) and search for "Galaxy" – look for a room named Galaxy or Galaxy Lobby (often with an address like #galaxyproject:matrix.org).
+  * Alternatively, you can use a direct Matrix link if provided, which will prompt your app to join the room.
+
+* Continue to join the room.
```

Comment:
should we add some direct links here to some of the Galaxy channels? at least the GTN and Galaxy Lobby, but maybe also some of the community channels, what do you think?

Reply:
@shiltemann yes - excellent idea. I can add some now.